app/pb_kits/playbook/pb_list_checkbox/_list_checkbox.jsx

```diff
@@ -0,0 +1,27 @@
+
+
+/* @flow */
+
+import React from 'react'
```

Comment:
Make sure to import: 

import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
import classnames from 'classnames'
import { globalProps } from '../utilities/globalProps.js'